### ISSUE TITLE
This repository is not REUSE-compliant

### ISSUE THREAD

**Issue by carmenbianca** [fsfe/reuse-website#5]
See title

From https://github.com/fsfe/reuse-tool/issues/29

**Comment by carmenbianca** [comment]
Recent changes have made this issue relevant again

**Comment by mxmehl** [comment]
Should have been fixed with b63535eda1dcc8989cdfcade294cc4bd5c2db342. However, with some warnings:

```
reuse._util - ERROR - Could not parse '[identifier]'
reuse.project - ERROR - reuse-docs/faq.md holds an SPDX expression that cannot be parsed, skipping the file
# SUMMARY

* Bad licenses:
* Missing licenses:
* Unused licenses:
* Used licenses: AGPL-3.0-or-later, Apache-2.0, CC-BY-SA-4.0, CC0-1.0, GPL-3.0-or-later, MIT, OFL-1.1
* Read errors: 0
* Files with copyright information: 67 / 67
* Files with license information: 67 / 67

Congratulations! Your project is compliant with version 3.0 of the REUSE Specification :-)
```
